product_docs/docs/edbcloud/beta/migration/index.mdx

```diff
@@ -0,0 +1,24 @@
+---
+title: Migrating databases to EDB Cloud
+---
+
+EDB provides migration tools to bring data from Oracle, PostgresSQL, and EDB Postgres Advanced Server databases into EDB Cloud. These tools include Migration Portal and Migration Toolkit for Oracle migrations. 
```

Comment:
How about pulling the last sentence of this doc up in to this one?

```suggestion
EDB provides migration tools to bring data from Oracle, PostgresSQL, and EDB Postgres Advanced Server databases into EDB Cloud. These tools include Migration Portal and Migration Toolkit for Oracle migrations. More sophisticated migration processes can use tools such as [Replication Server](../../../eprs/latest/) for ongoing migrations and [LiveCompare](../../../livecompare/latest/) for data comparisons. 
```

Reply:
Done